### DISCUSSION TITLE
Why does it randomize the requests order?

### DISCUSSION THREAD

**Discussion by Bayonadrian** [usebruno/bruno#2572] [Q&A]
I have just started with this app and I find it awesome, just yesterday I have started documenting a bunch of API's for a project, my idea is creating cruds ordered in a way similar to a pipeline:

![image](https://github.com/usebruno/bruno/assets/57043455/9d2fd069-bd03-4ce3-aa9b-f3ded91ae559)

This is to make everything run easier with runner without having to leave any record into the database. Then the pipeline is POST- GET - PUT - DELETE, yesterday everything was ok, but today since I have started I noticed something weird, and then I have noticed that whenever I run the app it randomizes my requests making impossible for the runner to run them in an appropriate way, is there any way to deal with this issue?

Keep pending on feedback
Thanks.

**Comment by Its-treason** [comment]
This should not happen. Its a bug intruduced in [`#2560`](https://github.com/usebruno/bruno/pull/2560/files#diff-e9fdfafb90e1be9d45a91eab5a850807e14d9fce3ae51c51e3d69b3e4a5a3300R168), see: https://github.com/usebruno/bruno/issues/2573 for more details. I already opened a PR to fix it: https://github.com/usebruno/bruno/pull/2574
